fieldpath/fieldpath-gen/template.go

```diff
@@ -1121,10 +1121,11 @@ func (this *{{.CName}}SinglePath) UnmarshalFirst(buf []byte) ({{.ReturnType}}, e
 				endOfs[position] = offset + int(length)
 			}
 		} else {
-			offset, err = skip(buf, offset, wireType)
+			newOffset, err := skip(buf, offset, wireType)
```

Comment:
I would like not to have any workarounds for false positives.
If this is the case, then this is exactly what I would like to avoid.